elastic/kibana
# [Uptime] Test now mode while adding/editing monitors

## Summary

Fixes https://github.com/elastic/uptime/issues/437

Add Test now mode  while adding/editing monitors

For browser monitors, it uses existing step list to display results inline.


### Testing

1. Add browser and simple monitors and as part of workflow, try testing those monitors using Test now button and verify the results in step list and ping list.
2. Make sure proper states are reflected with in progress, complete and pending state.

![image](https://user-images.githubusercontent.com/3505601/149354878-ef8638bb-d453-4664-974b-13fd6e662355.png)


For simple monitors it uses ping list to display results

![image](https://user-images.githubusercontent.com/3505601/149410726-f41079e2-f91c-4b95-a212-257d8777e977.png)


Will display error in case failed to push to service

![image](https://user-images.githubusercontent.com/3505601/149823797-24b6c8ab-6262-4d9d-b719-8d9d989b7a5b.png)
